documentation/specifications/common/authenticated_data_structures.md

```diff
@@ -288,7 +288,7 @@ The previous section described Merkle accumulator and sparse Merkle tree that fo
 
 The ledger state represents the ground truth about the Aptos ecosystem, including the quantity of coins held by each user at a given version. Each validator must know the ledger state at the latest version in order to execute new transactions. Specifically, the ledger state is a map from `AccountAddress` to a binary blob `AccountStateBlob` that represents the state of an account.
```

Comment:
Will update this as this doesn't hold true after the fine-grained storage anymore.